src/pages/register/register.html

```diff
@@ -89,12 +93,12 @@
         <p class="invalid-text">Passwords do not match</p>
       </ion-item>
 
-      <button id="register-button"color="secondary" ion-button [disabled]="!validate.valid">REGISTER </button>
+      <button id="register-button"color="secondary" ion-button [disabled]="!validate.valid" type="submit">REGISTER</button>
```

Comment:
Nice, I like the disabled functionality when the form is not valid. Let's re-emphasize in class this design pattern & explain its usefulness.